packages/assets-controllers/src/TokensController.ts

```diff
@@ -806,6 +830,12 @@ export class TokensController extends BaseController<
       throw rpcErrors.invalidParams(`Invalid address "${asset.address}"`);
     }
 
+    // Validate if account is an evm account
+    const selectedAccount = this.messagingSystem.call(
+      'AccountsController:getAccount',
+      this.#selectedAccountId,
+    );
```

Comment:
The comment indicates that there's a validation being done here but from the code it's not clear how. What should `getAccount` return if it's not an EVM account?

Reply:
This can be removed, the comment was from an earlier iteration where the selectedAccount could be a non evm account